frontend/src/queries/latest-versions.json

```diff
@@ -1 +1,6 @@
-{ "//": "THIS FILE IS AUTOGENERATED. DO NOT EDIT MANUALLY. To update: pnpm run schema:build", "RetentionQuery": 2 }
+{
```

Comment:
@thmsobrmlr tagging you because it looks like this is changing based on your PR from yesterday: https://github.com/PostHog/posthog/pull/35559. I guess when I ran `pnpm build:products` it regenerated this file which is causing some of the tests to fail. I've fixed everything but just so you have context